doc/manual/source_localization/forward.rst

```diff
@@ -739,50 +592,13 @@ point lines, consisting of seven numbers:
     allows the implementation of curved coils and coils with the gradiometer
     loops tilted with respect to each other.
 
-.. _BJEHHJIJ:
-
-Creating the coil definition file
-=================================
-
-The standard coil definition file $MNE_ROOT/share/mne/coil_def.dat
-is included with the MNE software package. The coil definition file
-can be recreated with the utility mne_list_coil_def
-as follows:
-
-mne_list_coil_def --out $MNE_ROOT/share/mne/coil_def.dat
-
 .. _CHDDIBAH:
 
 Computing the forward solution
 ##############################
 
-Purpose
-=======
-
-Instead of using the convenience script :ref:`mne_do_forward_solution` it
-is also possible to invoke the forward solution computation program :ref:`mne_forward_solution` directly.
-In this approach, the convenience of the automatic file naming conventions
-present in :ref:`mne_do_forward_solution` are
-lost. However, there are some special-purpose options available
-in :ref:`mne_forward_solution` only.
-Please refer to :ref:`BABCHEJD` for information on :ref:`mne_do_forward_solution`.
-See :ref:`mne_forward_solution` for command-line options.
-
-Implementation of software gradient compensation
```

Comment:
Shouldn't this be in **io.html** ?

Reply:
Then I would add a forward example with a CTF type, and apply the compensation and discuss it there.

Reply:
WDYT?

Reply:
@agramfort https://martinos.org/mne/dev/auto_tutorials/plot_brainstorm_phantom_ctf.html?highlight=apply_gradient_compensation